ScreensaverHero.hx

```diff
@@ -0,0 +1,75 @@
+package;
+
+    import flixel.FlxSprite;
+    import flixel.util.FlxColor;
+    import flixel.FlxG;
+
+
+    class ScreensaverHero extends FlxSprite {
+        
+        var hero_width:Int = 30;
+        var hero_height:Int = 30;
+        var blue:FlxColor = 0xFF1500FF;
+        var yellow:FlxColor = 0xFFFFFB00;
+        var green:FlxColor = 0xFF16B900;
+        var red:FlxColor = 0xFFFF0000;
+        var min_speed:Int = -75;
+        var max_speed:Int = 75;
+
+        public function new(x:Int = 0, y:Int = 0){
+            super(x,y);
+            super.makeGraphic(hero_height, hero_width);
+            this.velocity.x = FlxG.random.float(min_speed, max_speed);
+            this.velocity.y = FlxG.random.float(min_speed, max_speed);
```

Comment:
Nice work moving this into the constructor and leveraging variables to improve readability 👍